squad/core/tasks/__init__.py

```diff
@@ -151,16 +153,17 @@ def __call__(self, version, environment_slug, metadata_file=None, metrics_file=N
 
         testrun = build.test_runs.create(
             environment=environment,
-            tests_file=tests_file,
-            metrics_file=metrics_file,
-            log_file=log_file,
             metadata_file=metadata_file,
             completed=completed,
             **metadata_fields
         )
+        testrun.tests_file_storage.save("testrun/%s/tests_file" % testrun.pk, File(StringIO(tests_file)))
```

Comment:
Do you want to keep 
```python
tests_file=tests_file,
metrics_file=metrics_file,	
log_file=log_file,
```
?

This way we can rollout this patch and have data being saved to DB and S3 at the same time

Reply:
the idea is that we merge #849 first. Roll it out and migrate the data. Only then merge #850 which adds implementation for using the files from the storage. Lastly I will prepare a patch removing the old fields.